doc/user/content/ops/deployment.md

```diff
@@ -15,14 +15,14 @@ issue](https://github.com/MaterializeInc/materialize/issues/new?labels=C-feature
 Materialize stores the majority of its state in memory, and works best when the
 streamed data can be reduced in some way. For example, if you know that only a
 subset of your rows and columns are relevant for your queries, it helps to avoid
-materializing sources or views until you've expressed this to the system.
+materializing sources or views until you've expressed this to the system by manually creating indexes on the relevant columns.
```

Comment:
You don't need to manually create indexes. You can also "communicate to the system" by choosing between `CREATE VIEW` and `CREATE MATERIALIZED VIEW`. I'm a big fan of manually creating indexes, but I know that others are not.

Reply:
Reverted change.